src/interactions/User/AsEndUser.js

```diff
@@ -51,6 +54,28 @@ export class AsEndUser {
     return this.isLoggedIn() &&
            !!_find(this.user.notifications, {isRead: false})
   }
+
+  /**
+   * Returns a mr-text-* color for the user's displayName
+   */
+  colorCode() {
+    return mapColors(this.user.osmProfile.displayName)
+  }
+}
+
+/**
+ * Returns a mr-text-* color based off a hash of the username.
+ */
+export function mapColors(username) {
+  if (!username) return ""
+
+  return USER_COLOR_MAP[Math.abs(hashCode(username) % 6)]
```

Comment:
Can you please replace `6` with the length of the color map?

Reply:
Also, I guess it doesn't strictly matter, but don't you want to abs the hashCode instead of the result of the mod?